homeassistant/components/cast/translations/readme.md

```diff
@@ -0,0 +1,2 @@
+# Translations
+Our translations are managed via [Lokalise.](https://developers.home-assistant.io/docs/translations/)
```

Comment:
These will be shipped with all our distributions as well.

We should not add these. Our developer documentation provides documentation on how to contribute to this repository.